Code/pyproject.toml

```
@@ -48,6 +48,22 @@ selenium = "==4.37.0"
 stripe = "==13.0.1"
 
 
+# AI agents / LangGraph stack
+langchain = "*"
```

All of these packages need to be exactly versioned like the rest of our packages to keep our application stable as it builds on different machines. The way it is in there now says any version of the packages are accepted which is not how we want to do it.

You should be able to get a list of the versions of your packages that you have installed with poetry. Then just write them in following the format of the other packages versions. To lock down the versions.

```suggestion
langchain = "==INSERT.YOUR.PACKAGE_VERSION"
```